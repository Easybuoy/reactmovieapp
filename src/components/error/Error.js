import React, { Component } from 'react';
import Button from '@material-ui/core/Button';
import { Fa } from 'mdbreact';



class Error extends Component {
    constructor(props) {
        super(props);
        this.state = {
            error: 0
        };
        // this.onClick = this.onClick.bind(this);
    }

    reload(){

        // this.setState({error: 1})

        if(this.state.error === 1){
         window.location.reload();

        }
    }

    render() {
      
      return (
        <div>
          {/* <h1>{this.props.errorMessage} </h1> */}
          <div className="container">
            
            {/* <Button className="primary-color"> {this.props.buttonText}</a></Button> */}
            <a href='.'>
              <Button style={{'color': 'white'}} variant="contained" className="danger-color">
                <Fa icon="refresh" className="fav" />
              Reload
              
              </Button>
            </a>
          </div>
          <br />
  
        </div>
      );
    }
  }
  
  export default Error;
  