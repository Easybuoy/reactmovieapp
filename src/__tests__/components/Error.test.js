/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import Error from '../../components/error/Error';

describe('<Error />', () => {

  it('renders the Error component correctly', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
});
})

