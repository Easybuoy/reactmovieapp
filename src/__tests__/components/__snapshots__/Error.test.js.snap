// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> renders the Error component correctly 1`] = `ShallowWrapper {}`;
